### DISCUSSION TITLE
Unable to import a specific device type

### DISCUSSION THREAD

**Discussion by ouberlord** [netbox-community/netbox#15182] [Help Wanted!] [answered]
I am having the one of the oddest problems. I need to import a new device type into our Netbox environment (v3.7.2). I can import others no problem. However, when I attempt to import this one it is not successful. 

If I copy and paste the YAML file into the "Direct Import" tab it does not work, regardless of what the settings for Format or CSV Delimiter are set to. When I click Submit the screen flashes but stays on the import page, and there aren't any errors that appear.

If I attempt via the "Upload File" method I get the same behavior.

It is bizarre, because the file in question is a copy of another one that did import successfully, I just changed the model, slug, part_number, and a few of the ports.

I'm attaching the file in question; all I did was change its extension to txt (from yaml).

Anyone have tips on how I can troubleshoot this?

[AP7868.txt](https://github.com/netbox-community/netbox/files/14315484/AP7868.txt)

**Comment by candlerb** [comment, marked as answer]
Divide-and-conquer: keep removing bits from the end until it works.

It seems you have two entries with name "Bank2-C13-23"

**Comment by ouberlord** [comment]
I am legitimately impressed at your observational skills to notice that, because I absolutely did not. I corrected that, and the data imported without issue. Thank you!

**Comment by candlerb** [comment]
FYI, data validation issue was raised in #15185 and fixed in Netbox v3.7.3
